web/docs/tql2/operators/chart_area.md

```diff
@@ -0,0 +1,56 @@
+# chart_area
+
+Chart values on a X-Y area chart.
+
+```tql
+chart_area x=any, y=any, [x_min=any, x_max=any, resolution=duration, x_log=bool, y_log=bool, group=any, position=string]
+```
+
+## Description
+
+Chart values on X-Y area chart.
+
+### `x = any`
+
+The values to plot on the X-axis.
+
+### `y = any`
+
+The values to plot on the Y-axes. The values are grouped for each unique `x`
+with the given aggregation if specified.
+
+If specified as a record, the field names are used to label the Y-axes.
```

Comment:
There is only one y-axis, right? The field names are used to label the individual lines/areas.

Reply:
True, is there a standard term I can use here? Maybe `series`?

Reply:
Maybe "groups"? I know that there is a parameter with that name, but aren't they effectively two ways to do the same? So maybe using the same name is appropriate then.